clusterloader2/api/types.go

```diff
@@ -54,20 +54,41 @@ type Config struct {
 	ChaosMonkey ChaosMonkeyConfig `json:"chaosMonkey"`
 }
 
-// Step represents encapsulation of some actions. These actions could be
-// object declarations or measurement usages.
-// Exactly one field (Phases or Measurements) should be non-empty.
+// Step represents a unit of work in ClusterLoader2. It can be either:
+// - a collection of measurements,
+// - a collection of phases,
+// - a group of other steps (module).
+// Exactly one field (Phases or Measurements or Module) should be non-empty.
 type Step struct {
 	// Phases is a collection of declarative definitions of objects.
 	// Phases will be executed in parallel.
 	Phases []Phase `json:"phases"`
 	// Measurements is a collection of parallel measurement calls.
 	Measurements []Measurement `json:"measurements"`
-	// Name is an optional name for given step. If name is set,
-	// timer will be run for the step execution.
+	// Module is a group of steps, usually defined in a separate file.
```

Comment:
Nit: `sequence` would probably be a better word here than `group`.

Reply:
Done.